grub 2.12 introduced the `--is-supported` argument for fwsetup, and runs it for
efi systems to determine whether to show the uefi-firmware menu item.

The problem is that grub 2.06 does not support the `--is-supported` flag yet,
and calling it just crashes grub, causing the machine to reboot.

Showing a menu entry for uefi-firmware that may not work is infinitely better
than crashing and preventing the machine to boot, so remove the condition and
always show the menu entry.

See also: https://bugs.archlinux.org/task/75701

diff --git a/util/grub.d/30_uefi-firmware.in b/util/grub.d/30_uefi-firmware.in
index 1c2365d..b6041b5 100644
--- a/util/grub.d/30_uefi-firmware.in
+++ b/util/grub.d/30_uefi-firmware.in
@@ -32,11 +32,8 @@ gettext_printf "Adding boot menu entry for UEFI Firmware Settings ...\n" >&2
 
 cat << EOF
 if [ "\$grub_platform" = "efi" ]; then
-	fwsetup --is-supported
-	if [ "\$?" = 0 ]; then
-		menuentry '$LABEL' \$menuentry_id_option 'uefi-firmware' {
-			fwsetup
-		}
-	fi
+	menuentry '$LABEL' \$menuentry_id_option 'uefi-firmware' {
+		fwsetup
+	}
 fi
 EOF
